tiledb/sm/storage_manager/storage_manager.h

```diff
@@ -687,6 +687,19 @@ class StorageManager {
   Status get_latest_array_schema_uri(
       const URI& array_uri, URI* schema_uri) const;
 
+  /**
+   * Loads the schema of a schema uri from persistent storage into memory.
+   *
+   * @param array_schema_uri The URI path of the array.
+   * @param encryption_key The encryption key to use.
+   * @param array_schema The array schema to be retrieved.
+   * @return Status
+   */
+  Status load_array_schema_uri(
+      const URI& array_uri,
+      const EncryptionKey& encryption_key,
```

Comment:
Please coordinate with @bekadavis9  on which PR will be merged first (this one or https://github.com/TileDB-Inc/TileDB/pull/2412), and the other needs to be adjusted.

Reply:
Discussed that with Beka and Seth. It seems that we are only deprecating encryption_key in c-api at the mean time.